include/deal.II/lac/sparse_vanka.templates.h

```diff
@@ -113,13 +111,15 @@ SparseVanka<number>::compute_inverses()
 #else
   const size_type n_inverses =
     std::count(selected->begin(), selected->end(), true);
-
-  const size_type n_inverses_per_thread =
-    std::max(n_inverses / n_threads, static_cast<size_type>(1U));
+  // somewhat arbitrarily set up an equal number of tasks as we have threads
```

Comment:
you don't have to fix this, but I would guess having more tasks makes sense.

Reply:
I agree, but I am leaving that for a followup PR.